job/models.py

```diff
@@ -3579,6 +3575,16 @@ class ExistingBoilerOutputs(BaseModel, models.Model):
         default = list,
     )
 
+    year_one_thermal_to_tes_series_mmbtu_per_hour = ArrayField(
```

Comment:
@rathod-b do you know why there are two `thermal_to_tes_series_mmbtu_per_hour` outputs for ExistingBoiler?